### PR TITLE
Turkish translation

### PR DESCRIPTION
**Pull request by emrgncr**

### CODE REVIEW

**Comment by rottenpants466** [comment]
You did the same mistake i did when translating to my language :|

You need to delete `type="unfinished"`

Instead of this:
`<translation type="unfinished">Pil şarjı:</translation>`

Change it to this:
`<translation>Pil şarjı:</translation>`

**Comment by emrgncr** [comment]
Oh thanks.

**Comment by dmitry-s93** [comment]
Hello.

There are untranslated lines in the file. If in your language they are written the same as in the source language, then just copy from source.

**Comment by emrgncr** [comment]
Hello, I copied from source for untranslated lines and I removed the "unfinished" tags.

File: src/i18n/MControlCenter_tr.ts [hunk 1]
```diff
@@ -0,0 +1,230 @@
...
+        <translation>Performans ile pil ömrünü dengeleyen bir mod</translation>
+    </message>
+    <message>
+        <source>Silent</source>
+        <translation>Sessiz</translation>
+    </message>
+    <message>
+        <source>Super Battery</source>
+        <translatorcomment>I don&apos;t know what this is</translatorcomment>
+        <translation>Super Battery</translation>
```
Comment (dmitry-s93) [inline src/i18n/MControlCenter_tr.ts:193]:
```suggestion
        <translation type="unfinished">Super Battery</translation>
```

In this case, it's better to leave unfinished

File: src/i18n/MControlCenter_tr.ts [hunk 2]
```diff
@@ -0,0 +1,230 @@
...
+    </message>
+    <message>
+        <source>EC Version:</source>
+        <translatorcomment>don&apos;t know how to translate this</translatorcomment>
+        <translation>EC Version:</translation>
+    </message>
+    <message>
+        <source>EC Build:</source>
+        <translatorcomment>don&apos;t know how to translate this</translatorcomment>
+        <translation>EC Build:</translation>
```
Comment (dmitry-s93) [inline src/i18n/MControlCenter_tr.ts:18]:
```suggestion
        <translation type="unfinished">EC Build:</translation>
```

In this case, it's better to leave unfinished

**Pull request merged**